proto/viam/provisioning/provisioning.proto

```diff
@@ -0,0 +1,90 @@
+syntax = "proto3";
+
+package viam.provisioning.v1;
+
+option go_package = "go.viam.com/api/provisioning/v1";
+
+// ProvisionService is used by viam-agent to retrieve device and robot info.
+service ProvisionService {
+  // GetDeviceStatus is for retrieving the status of the device including networking.
+  rpc GetDeviceStatus(GetDeviceStatusRequest) returns (GetDeviceStatusResponse);
+
+  // SetWifiCredentials is to set the wifi credentials.
+  rpc SetWifiCredentials(SetWifiCredentialsRequest) returns (SetWifiCredentialsResponse);
+
+  // SetRobotCredentials is to set the robot credentials.
+  rpc SetRobotCredentials(SetRobotCredentialsRequest) returns (SetRobotCredentialsResponse); 
+
+  // GetVisibleWifiNetworks is to retrieve the list of wifi networks that are visible to the device.
+  rpc GetVisibleWifiNetworks(GetVisibleWifiNetworksRequest) returns (GetVisibleWifiNetworksResponse); 
```

Comment:
Should we call this just GetVisibleNetworks?

Reply:
GetNetworkList?